src/ff/mod.rs

```diff
@@ -3,13 +3,16 @@
 // This is where we store arithmetic shared secret data models.
 
 mod field;
+mod galois_field;
 mod prime_field;
 
-pub use field::{BinaryField, Field, FieldType, Int};
+pub use field::{Field, FieldType, Int};
+pub use galois_field::{Gf40Bit, Gf8Bit};
 pub use prime_field::{Fp31, Fp32BitPrime};
 
 use crate::secret_sharing::SharedValue;
-use std::ops::{Add, AddAssign, Mul, MulAssign, Neg, Sub, SubAssign};
+use generic_array::{ArrayLength, GenericArray};
+use std::ops::{Add, AddAssign, BitXor, BitXorAssign, Index, Mul, MulAssign, Neg, Sub, SubAssign};
```

Comment:
Why do you need to import these now?

Reply:
Just moving what was on line 66-67 to the top.